package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-brightspace",
-  "version": "0.5.1",
+  "version": "0.5.2",
```

Comment:
This will probably break anyone's build who's using it, although a quick code search says that's only our core repo right now, so 🤷‍♂️

Reply:
Yeah I figured we were probably fine.  If we did a major bump... I think we'd need to update every single repo in BSI